Lib/fontTools/feaLib/parser.py

```diff
@@ -10,13 +10,15 @@
 import re
 
 
+
```

Comment:
no, I asked you to not remove line 8 of lexer.py; but you kept the removal, and instead added a new empty line in `parser.py`...
Anyway, not worth arguing about these minor things :)
I simply don't want random whitespace changes to distract from the content of the patch.